lib/Db/StepMapper.php

```diff
@@ -51,6 +52,24 @@ public function find($documentId, $fromVersion, $lastAckedVersion = null) {
 		return $this->findEntities($qb);
 	}
 
+	public function getLatestVersion($documentId): ?string {
```

Comment:
Maybe we can return an int here?

```suggestion
	public function getLatestVersion($documentId): ?int {
```